apps/voting-disputable/package.json

```diff
@@ -24,16 +25,17 @@
     "apm:publish:patch": "aragon apm publish patch --files public/ --prepublish-script apm:prepublish"
   },
   "dependencies": {
-    "@aragon/os": "5.0.0-beta.2",
+    "@aragon/os": "5.0.0-beta.5",
     "@aragon/apps-agreement": "^0.1.0-beta.0"
   },
   "devDependencies": {
     "@aragon/apps-shared-migrations": "1.0.0",
-    "@aragon/cli": "^7.1.3",
+    "@aragon/cli": "^7.0.0",
     "@aragon/contract-helpers-test": "^0.1.0-rc.2",
     "@aragon/minime": "^1.0.0",
-    "@aragon/staking": "^0.2.3",
-    "@aragon/truffle-config-v5": "^1.0.0",
+    "@aragon/staking": "^0.2.4",
+    "@aragon/truffle-config-v5": "^1.0.1",
+    "chai": "^4.2.0",
```

Comment:
Curious about this, why do we need it now here explicitly?

Reply:
it's a peer dependency of the contract-helpers-test library now :)